src/components/App.jsx

```diff
@@ -48,35 +48,37 @@ export const DumbApp = ({ breakpoints: { isDesktop } }) => {
           <Layout>
             <Sidebar />
             <Main>
-              <Content className="u-p-2">
-                <Switch>
-                  <RedirectIfExtensionInstalled
-                    path="/presentation"
-                    component={PresentationPage}
-                  />
-                  <RedirectIfExtensionInstalled
-                    path="/security"
-                    exact
-                    component={SecurityPage}
-                  />
-                  <RedirectIfExtensionInstalled
-                    path="/security/hint"
-                    exact
-                    component={HintPage}
-                  />
-                  <RedirectIfExtensionInstalled
-                    path="/installation"
-                    exact
-                    component={InstallationPage}
-                  />
-                  <Route
-                    path="/installation/installed"
-                    exact
-                    component={InstalledPage}
-                  />
-                  <Redirect from="/" to="/presentation" />
-                  <Redirect from="*" to="/presentation" />
-                </Switch>
+              <Content>
+                <div className="u-p-2 u-mt-auto u-mb-auto">
```

Comment:
Are we sure we do not want a margin-bottom here ? I have a bit of trouble understanding why the password app should have a custom layout.

Reply:
The password app has its content vertically centered. That's why we have auto vertical magins.

Reply:
Are you sure that the password app will have all its pages with vertically centered content ? I think this is a coincidence that all the pages, as of now, are vertically centered. IMHO, the padding and vertical centering should not be hard coded in the general layout of the app, but preferably in a wrapper for all views.

Reply:
I see your point, you're right. I'm doing this change right now.

Reply:
See https://github.com/cozy/cozy-passwords/pull/15/commits/597ac9498d8cc098e4e7150618ef28fdcd8a074e

Reply:
Thanks !